editor/translations/extract.py

```diff
@@ -109,10 +109,10 @@ class ExtractType(enum.IntEnum):
     re.compile(r'TTR\("(?P<message>([^"\\]|\\.)*)"(, "(?P<context>([^"\\]|\\.)*)")?\)'): ExtractType.TEXT,
     re.compile(r'TTRC\("(?P<message>([^"\\]|\\.)*)"\)'): ExtractType.TEXT,
     re.compile(
-        r'TTRN\("(?P<message>([^"\\]|\\.)*)", "(?P<plural_message>([^"\\]|\\.)*)",[^,)]+?(, "(?P<context>([^"\\]|\\.)*)")?\)'
+        r'TTRN\([^,]+?, "(?P<message>([^"\\]|\\.)*)", "(?P<plural_message>([^"\\]|\\.)*)"(, "(?P<context>([^"\\]|\\.)*)")?\)'
```

Comment:
For `TTRN()`, `RTRN()`, `DTRN()` I think `plural_message` should be left required?